tests/integration/targets/mso_version/tasks/connection.yml

```diff
@@ -0,0 +1,144 @@
+# Test code for the MSO modules
+# Copyright: (c) 2023, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Set vars
+  ansible.builtin.set_fact:
+    plugin_query: &plugin_query
+      state: query
+      output_level: debug
+
+- name: Set ansible_connection to ansible.netcommon.httpapi with a wrong domain
+  ansible.builtin.set_fact:
+    ansible_connection: ansible.netcommon.httpapi
+    ansible_httpapi_login_domain: wrong_domain_from_plugin
```

Comment:
I'm not sure if the login_domain actually has the value "wrong_domain_from_plugin" after this setting.
Can we make sure that login_domain is indeed set here?

Reply:
Resolving this as it is set